aiida/cmdline/commands/cmd_profile.py

```diff
@@ -92,22 +102,45 @@ def profile_setdefault(profile):
     help='Include deletion of associated file repository.'
 )
 @arguments.PROFILES(required=True)
-def profile_delete(force, include_config, include_db, include_repository, profiles):
-    """
-    Delete one or more profiles.
+def profile_delete(force, include_config, include_database, include_database_user, include_repository, profiles):
+    """Delete one or more profiles.
 
-    You can specify more profile names (separated by spaces).
-    These will be removed from the aiida config file,
-    and the associated databases and file repositories will also be removed.
+    The PROFILES argument takes one or multiple profile names that will be deleted. Deletion here means that the profile
+    will be removed including its file repository and database. The various options can be used to control which parts
+    of the profile are deleted.
     """
-    from aiida.manage.configuration.setup import delete_profile
+    if not include_config:
+        echo.echo_deprecated('the `--skip-config` option is deprecated and is no longer respected.')
 
     for profile in profiles:
-        echo.echo_info(f"Deleting profile '{profile.name}'")
-        delete_profile(
-            profile,
-            non_interactive=force,
-            include_db=include_db,
-            include_repository=include_repository,
-            include_config=include_config
+
+        includes = {
+            'database': include_database,
+            'database user': include_database_user,
+            'file repository': include_repository
+        }
+
+        if not all(includes.values()):
+            message_suffix = ', excluding: '
+            excludes = []
+            for include_label, include in includes.items():
+                if not include:
+                    excludes.append(include_label)
```

Comment:
Or just
```suggestion
            excludes = [ label for label in includes if not includes[label] ]
```

Reply:
Hah, of course. I am definitely including this, it's way better. Thanks :+1: